relay-config/src/config.rs

```diff
@@ -354,6 +354,8 @@ pub struct Relay {
     pub tls_identity_path: Option<PathBuf>,
     /// Password for the PKCS12 archive.
     pub tls_identity_password: Option<String>,
+    /// Events for these projects will not be forwarded
+    pub _skip_projects: Option<HashSet<u64>>,
```

Comment:
Please use a `BTreeSet`, that should be a bit faster for "small" sets.

Reply:
👍